test/sub_server/test_server_asyncio.py

```diff
@@ -125,7 +125,7 @@ async def _setup_teardown(self):
 
         # teardown
         if self.server is not None:
-            await self.server.server_close()
+            await self.server.shutdown()
```

Comment:
We need to test both shutdown and server_close as they are both part of the API.

Reply:
Please leave one for server_close and the rest as shutdown.

Reply:
see comment above regarding 0071ac11d0e050b2e168b86e6cde42898855e1a8